README.md

```diff
@@ -104,58 +114,120 @@ Contains Non-ASCII Characters: 12
 clkhash will garble personally identifiable information (PII) in a way that it can be used for linkage later on. The CODI PPRL process garbles information a number of different ways. The `garble.py` script will manage executing clkhash multiple times and package the information for transmission to the DCC.
 
 `garble.py` requires 3 different inputs:
-1. The location of a directory of clkhash linkage schema files
-1. The salt value to use in the garbling process
 1. The location of a CSV file containing the PII to garble
+1. The location of a directory of clkhash linkage schema files
+1. The location of a secret file to use in the garbling process (you have to generate this file yourself - if you are not sure how, you may find the `testing-and-tuning/generate_secret.py` script useful)
```

Comment:
Rather than the parenthetical, which gently suggests I'm missing something, maybe just specify something like "this should be a text file containing a single hexadecimal string of at least 128 characters; the `testing-and-tuning/generate_secret.py` script will create this for you if require it, e.g.:

```python testing-and-tuning/generate_secret.py```

This should create a new file called `deidentification_secret.txt` in your root directory."

Reply:
And if this secret file is to be generated for data owners or otherwise shared, should probably mention that here too.